Sources/TuistSupport/Utils/Environment.swift

```diff
@@ -91,10 +90,23 @@ public class Environment: Environmenting {
 
     /// Returns true if the output of Tuist should be coloured.
     public var shouldOutputBeColoured: Bool {
-        if let coloredOutput = ProcessInfo.processInfo.environment[Constants.EnvironmentVariables.colouredOutput] {
-            return Constants.trueValues.contains(coloredOutput)
+        let noColor = if let noColorEnvVariable = ProcessInfo.processInfo.environment["NO_COLOR"] {
+            Constants.trueValues.contains(noColorEnvVariable)
+        } else {
+            false
+        }
+        let ciColorForce = if let ciColorForceEnvVariable = ProcessInfo.processInfo.environment["CLICOLOR_FORCE"] {
+            Constants.trueValues.contains(ciColorForceEnvVariable)
+        } else {
+            false
+        }
+        if noColor {
+            return false
+        } else if ciColorForce {
+            return true
```

Comment:
I think we should just check whether the variables are set, regardless of their value, isn't it?
Otherwise, I don't get why we would need 2 of them

Reply:
According to that page we can just check for the presence. However, I'm inclined to keep the truthiness check because something like NO_COLOR=0 or `NO_COLOR=foo` doesn't make sense.